README.md

```diff
@@ -5,12 +5,12 @@
 `npm install`
 
 To build css for prod
-`npm run build-prod`
+`npm run build`
 
-To build a preview for test (watch included)
-`npm run build-test`
+To build a preview for test
+`npm run watch`
 
-Built with postcss: https://www.npmjs.com/package/postcss-cli
+Built with [Postcss](Built with postcss: https://www.npmjs.com/package/postcss-cli).
```

Comment:
This can't work in Markdown, it renders a broken link. You mean:
```
Built with [Postcss](https://www.npmjs.com/package/postcss-cli).
```

Reply:
fixed ... I had just copy paste your line without checking ... sorry ^^